repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -84,6 +84,7 @@ class SparkILoop(in0: Option[BufferedReader], out: JPrintWriter)
     echo(welcomeMsg)
     echo("Type in expressions to have them evaluated.")
     echo("Type :help for more information.")
+    echo("Type ctrl + c to exit.")
```

Comment:
Ctrl-D or `:quit` exits. Scala shell doesn't print this and I think ctrl-D is widely understood to exit processes like this. Ctrl-C is an unnecessarily hard exit but also works. Is it really unclear to anyone?